### PR TITLE
Use bs4_book template

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -21,12 +22,20 @@ Imports:
     usethis,
     withr
 Suggests:
+    bookdown,
```
Comment (hadley) [inline DESCRIPTION:25]: FWIW the distinction I use for imports vs suggests for books is that imports is need to run the code in the book; suggests is needed to build the book itself

File: _output.yml [hunk 2]
```diff
@@ -1,3 +1,12 @@
+bookdown::bs4_book:
+  theme:
+    primary: "#982a31"
```
Comment (hadley) [inline _output.yml:3]: I don't know if we need to formally record this anywhere but these colours were produced by taking O'Reilly red (#BF353F) and setting chroma to 80 and lightning to 35 for the primary colour, and chroma to 5 and lightness to 14 for the text colour.

File: DESCRIPTION [hunk 3]
```diff
@@ -22,19 +24,18 @@ Imports:
...
+    downlit,
+    jsonlite,
     parsermd,
-    pkgload
-Remotes:  
+    pkgload,
+    xml2
+Remotes: 
     r-lib/devtools,
     r-lib/gert,
```
Comment (jennybc) [inline DESCRIPTION:36]: We are probably fine with CRAN gert now

**Comment by hadley** [comment]
[Whole game](https://5f9f3ad97330086a15debcea--r-pkgs.netlify.app/whole-game.html) looks good to me (although I don't see any colour), so I think this is good to merge. As long as any problems are spotted in the next ~2 weeks I should be able to fix quickly.

**Comment by jennybc** [comment]
I see the ANSI escapes:

Screen Shot 2020-11-02 at 6 56 37 AM

**Comment by hadley** [comment]
Ok, I see it too at https://5f9f504f2fba21013795d11a--r-pkgs.netlify.app/whole-game.html#create_package

**Pull request merged**